docs/source/guides/user/chapters/requirements.rst

```diff
@@ -0,0 +1,71 @@
+.. _managing-requirements:
+
+Managing Requirements
+=====================
+
+.. note:: Test Requirements are supported only on the nrunner runner. To use this
+          feature, remember to use `--test-runner=nrunner` argument.
+
+The test Requirements feature can fulfill the requirements a test may have.
+
+The test requirements are specified in the test definition and are fulfilled
+based on the supported requirement `type`.
+
+Test workflow with requirements
+-------------------------------
+
+When a requirement is defined for a test, it is marked as a dependency for that
+test. The test will wait for all the requirements to complete successfully before
+it is started.
+
+When any of the requirements defined on a test fails, the test is skipped.
+
+Defining a test requirement
+---------------------------
+
+A test requirement is described in the JSON format. Following is an example of
+a requirement of `type` `package`::
+
+        {"type": "package", "name": "hello"}
+
+To define a requirement for the test, use the test's docstring with the format of
+keywords `:avocado: requirement=`. The following example shows the same package
+requirement showed above inside a test docstring::
+
+        from avocado import Test
+
+
+        class PassTest(Test):
+            """
+            :avocado: requirement={"type": "package", "name": "hello"}
```

Comment:
It would be nice to have an example  with multiple requirements (or at least mention it is supported).